api/clients/v2/payload_disperser.go

```diff
@@ -270,12 +270,8 @@ func (pd *PayloadDisperser) buildEigenDACert(
 
 	timeoutCtx, cancel := context.WithTimeout(ctx, pd.config.ContractCallTimeout)
 	defer cancel()
-
-	// TODO (litt3): @reviewers I think this method (buildEigenDACert) is the main end user of GetNonSignerStakesAndSignature,
-	//  and we are just discarding the signedQuorumNumbers returned from GetNonSignerStakesAndSignature. IMO this points
-	//  to returning signedQuorumNumbers being the wrong API choice
-	nonSignerStakesAndSignature, _, err := pd.certVerifier.GetNonSignerStakesAndSignature(
-		timeoutCtx, blobStatusReply.GetSignedBatch())
+	nonSignerStakesAndSignature, err := pd.certVerifier.GetNonSignerStakesAndSignature(
+		timeoutCtx, certVerifierAddress, blobStatusReply.GetSignedBatch())
 	if err != nil {
 		return nil, fmt.Errorf("get non signer stake and signature: %w", err)
```

Comment:
@litt3 we can do this in another PR but would be good for now on to start adopting some standard error construction for eth_calls like we discussed during standup this week. Otherwise our clients don't know when they see an error that it might be coming from their own rpc provider, as opposed to being a bug on our side.

Reply:
My assumption has been that whatever `err` returned would be informative in some way, but I haven't tested that assumption. Agree, if the error returned from the eth call isn't informative enough, we need to find a way to add context